Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM cyberbotics/webots 	
```

Comment:
It's probably safer to specify a version here (so we are sure which one we are using).